designer/client/src/components/toolbars/scenarioActions/buttons/CustomActionButton.tsx

```diff
@@ -11,29 +11,23 @@ import UrlIcon from "../../../UrlIcon";
 type CustomActionProps = {
     action: CustomAction;
     processName: string;
-    processStatus: StatusType | null;
 } & ToolbarButtonProps;
 
 export default function CustomActionButton(props: CustomActionProps) {
-    const { action, processStatus, disabled, type } = props;
+    const { action, disabled, type } = props;
 
     const { t } = useTranslation();
 
     const icon = action.icon ? <UrlIcon src={action.icon} FallbackComponent={DefaultIcon} /> : <DefaultIcon />;
 
-    const statusName = processStatus?.name;
-    const available = !disabled && action.allowedStateStatusNames.includes(statusName);
-
-    const toolTip = available
-        ? null
-        : t("panels.actions.custom-action.tooltips.disabled", "Disabled for {{statusName}} status.", { statusName });
+    const toolTip = disabled ? t("panels.actions.custom-action.tooltips.disabled", "Disabled for current state.") : null;
```

Comment:
Why do we change the previous message? IMO From the user's perspective it sounds better to have the info about status in this tooltip

Reply:
Ok I've added some additional description in `Describe your changes` section. So I hope it brings some insight behind the change. And replying to this very question I removed  particular status name here  as I thought it can be little misleading as status of the button don't rely only on scenario status now.  Additionally this err message was the only reason to pass this status here as an function's argument.